server/scripts/database.js

```diff
@@ -168,8 +168,9 @@ module.exports.addLap = function() {
  */
 function jsonToMap(jsonObj) {
   const mapObj = Object.assign(columnMap);
+  const timestamp = (new Date(jsonObj['TimeStamp']).getTime()).toFixed(0)
```

Comment:
Are map keys case sensitive? If so I think this should be jsonObj['timestamp']

Reply:
In the json object that the Test Tool/Hermes sends, the values are cased differently. They become `lowercase` when entered to the database.